memcached/conf/memcached.properties

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2015 YCSB contributors. All rights reserved.
```

Comment:
Per discussion below, we're keeping it as `Copyright (c) 2015 YCSB contributors.` so yes, this is now (retroactively) correct.